sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md

```diff
@@ -2,6 +2,10 @@
 
 ## Current version
 ###  Feature Additions
+- Added support for Data Flow:
```

Comment:
You can update the version here now - add  "## Version 4.2.0"

Reply:
done